src/invidious/helpers/proxy.cr

```diff
@@ -236,7 +236,7 @@ def get_spys_proxies(country_code = "US")
     proxies << {ip: ip, port: port, score: score}
   end
 
-  proxies = proxies.sort_by { |proxy| proxy[:score] }.reverse
+  proxies = proxies.sort_by { |proxy| proxy[:score] }.reverse!
```

Comment:
this should also use `sort_by!` instead of `sort_by`. And also the `(&.[:score])` form, no?